serialized_redis/__init__.py

```diff
@@ -50,7 +52,7 @@ def serialize(self, value):
         return self.serialize_fn(value)
 
     def deserialize(self, value):
-        if value is None or value is '':
+        if value': #Im pretty sure that could be just if value
```

Comment:
I guess you meant `if not value` :)

Reply:
Woops! That is correct! My bad!

Reply:
However this pull requests might not be what everyone wants but is a good one for those who are playing around saving orderedDict for headers, cookies or whatever else it could be!

Reply:
Maybe, another way to implement it without breaking changes would be to add a `unpackb_kwargs` to `MsgpackSerializedRedis`.

Then you could use `MsgpackSerializedRedis` like this:

```
MsgpackSerializedRedis(host='localhost', port=6379, db=0, unpackb_kwargs={"object_pairs_hook": OrderedDict})
```
This would certainly be more flexible.
We could do the same with `packb_kwargs` and for `JSONSerializedRedis` and `PickleSerializedRedis`...

Reply:
I totally agree with that! Do also remember that I added `msgpack.packb` in line 490. I replaced dumps to packb instead :) Do you want to do the honor to do the changes? Just incase I don't break your beautiful code :D